jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -167,6 +167,66 @@ Provided in the output from `terraform apply`
+  * You may also create a nomad IAM user manually. The user will need an IAM policy with the following rules attached:
+[source, json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:CreateOrUpdateTags",
+                "autoscaling:UpdateAutoScalingGroup",
+                "autoscaling:TerminateInstanceInAutoScalingGroup"
+            ],
+            "Resource": "<<Your Autoscaling Group ARN>>"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeScalingActivities",
+                "autoscaling:DescribeAutoScalingGroups"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+. Set Nomad Auto-Scaler to `enabled`
+. Set Max Node Count* - This will overwrite what is currently set as the max for you ASG. It is recommended to keep this value and what was set in your terraform as the same.
+. Set Min Node Count* - This will overwrite what is currently set as the max for you ASG. It is recommended to keep this value and what was set in your terraform as the same.
+. Select cloud provider: `AWS EC2`
+. Add the region of the auto scaling group
+. Add the nomad autoscaler user's access key
+. Add the nomad autoscaler user's secret key
+. Add the name of the autoscaling Group your nomad clients were created in
+
+===== GCP
+. Create a service account for Nomad Autoscaler
+  * Our link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[nomad module] creates a service acount and outputs a file with the keys if you set variable `nomad_auto_scaler = true`. You may reference the examples in the link for more details. If you've already created the clients, you can simply update the variable and run `terraform apply`. The created user's key will be available in a file named `nomad-as-key.json`.
+  * You may also create a nomad gcp service account manually. The service account will need the role `compute.admin`.
+. Set Nomad Auto-Scaler to `enabled`
+. Set Maximum Node Count*
+. Set Minimum Node Count*
+. Select cloud provider: `Google Cloud Platform`
+. Add your Project ID
+. Add Managed Instance Group Name
+. Instance group type: link:https://cloud.google.com/compute/docs/instance-groups/#types_of_managed_instance_groups[Zonal or Regional].
+. JSON of GCP service account for Nomad Auto-Scaler
+
+```
+* The maximum and minimum nomad client count will overwrite the corresponding values set when you created your autoscaling group or managed instance group. It is recommended that you keep these values and those used in your terraform the same so that the 2 don't compete.
```

Comment:
```suggestion
* The maximum and minimum nomad client count will overwrite the corresponding values set when you created your autoscaling group or managed instance group. It is recommended that you keep these values and those used in your Terraform the same so that the two don't compete.
```